resources/views/hr/application/index.blade.php

```diff
@@ -9,10 +9,16 @@
         <div class="col-md-3">
             <h1>Applications</h1>
         </div>
-        <div class="input-group mb-3 col-md-6" style="display: flex;">
+        <div class="input-group mb-1 col-md-6" style="display: flex;">
             <form method="GET" action="/{{ Request::path() }}">
                 <input type="hidden" name="status" class="form-control" id="search"
                     value="{{ config('constants.hr.status.' . request("status") . '.label') }}">
+                <input type="hidden" name="start-year" class="form-control" id="search"
+                    value=@if(request()->has('start-year')){{request()->get('start-year')}}@endif>
```

Comment:
I think this should work
```suggestion
                    value={{ old('start-year', request()->get('start-year')) }}>
```